README.md

```diff
@@ -91,26 +91,39 @@ mvn spring-boot:stop
 # Make sure you are in the root of the repo
 # Run the command below to stop the previous instance and free up port 4120:
 
-mvn spring-boot:stop 
+mvn spring-boot:stop
 
 # You can also hit Ctrl-C from the shell window
 
-# Build the image and run the container using Docker 
+# Build the image and run the container using Docker
 
 # make sure you are in the root of the repo
 
 # this will use Azure CLI cached credentials
 
 # build the image
 
-docker build -t helium-dev .
+# IMPORTANT: In the following steps, you will be using the Dockerfile-Dev to build a developer
```

Comment:
should instruct to run `id` and if the uid != 1000 && gid != 1000, need to edit Dockerfile-Dev

this would be a good place to explain -v - what and why

Reply:
done.

Reply:
See above about finding the highest UID with nextuid